### PR TITLE
fix: use echart color config

### PR DESCRIPTION
**Pull request by rephus**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Description:

Refactor the way we set color on echarts (re: org colour palette) 

From @owlas :

> We don’t need this custom logic here to override the custom colour pallete on the series if the user hasn’t chosen a specific colour: [https://github.com/lightdash/lightdash/blob/4432de0ed51df08a59d8c9e23172991116a752[…]src/components/LightdashVisualization/VisualizationProvider.tsx](https://github.com/lightdash/lightdash/blob/4432de0ed51df08a59d8c9e23172991116a75223/packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx#L83-L85)
we can set our colour palette with a top level field: https://echarts.apache.org/en/option.html#color

It also fixes the list of default colors on `configuration`

### Preview:

image

### Scope of the changes (select all that apply):
- [x] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

File: packages/frontend/src/hooks/useEcharts.ts [hunk 1]
```diff
@@ -253,6 +260,10 @@ export const getEchartsSeries = (
                     tooltip: {
                         valueFormatter: valueFormatter(xField, yField, explore),
                     },
+                    color: series.color
```
Comment (owlas) [inline packages/frontend/src/hooks/useEcharts.ts:263]: @rephus I made a mistake before!

I meant to say we can set the colours at the very top level: https://github.com/lightdash/lightdash/blob/e8abb8a1cf163d07b0ae16ead2cc047a0c10b422/packages/frontend/src/hooks/useEcharts.ts#L486-L486

Then here we don't need any reference to the organization colours on the individual series.

Comment (owlas) [reply packages/frontend/src/hooks/useEcharts.ts:263]: See here: https://echarts.apache.org/en/option.html#color

**Review by owlas — CHANGES_REQUESTED** [review]
I suggested changing the wrong piece of code previously I'm so sorry!

I meant to point to this line where we can add a global configuration that applies to all series

File: packages/frontend/src/hooks/useEcharts.ts (hunk 1 above)
Comment (rephus) [reply packages/frontend/src/hooks/useEcharts.ts:263]: Right, sorry, I thought that was the right configuration. Is there a place were we set the global echart `setOption` config ?

Comment (owlas) [reply packages/frontend/src/hooks/useEcharts.ts:263]: Yes! @rephus you can find the actual Echarts call here and the options are passed directly as the output of the `useEcharts` hook: https://github.com/lightdash/lightdash/blob/a7f59f5dc2a3ac12a195fb5fbc8bc140741d77d2/packages/frontend/src/components/SimpleChart/index.tsx#L98-L111

**Pull request merged**